integration/angular/simple-message.ts

```diff
@@ -1,6 +1,21 @@
 import { Writer, Reader } from 'protobufjs/minimal';
 
 
+function getGlobalThis() {
+  if (typeof globalThis !== "undefined") {
+    return globalThis;
+  }
+  if (typeof self !== "undefined") {
+    return self;
+  }
+  if (typeof window !== "undefined") {
+    return window;
+  }
+  if (typeof global !== "undefined") {
+    return global;
+  }
```

Comment:
Would it be possible to write in a less verbose way? Like

```ts
function getGlobalThis() {
  if (typeof globalThis !== "undefined") return globalThis;
  if (typeof self !== "undefined") return self;
  if (typeof window !== "undefined") return window;
  if (typeof global !== "undefined") return global;
}
```

Or does the code generator not support this?

Reply:
You can add anything you like, but not using the nice `.beginControlFlow`/`.endControlFlow` methods.

Reply:
I don't have strong opinions here, just a slight tendency to avoid polluting people's code bases with many lines of boilerplate